pages/admin/lessons.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import { AdminLessonInfo } from '../../components/AdminLessonInfo'
+import { AdminLessonsSideBar } from '../../components/AdminLessonsSideBar'
+import LoadingSpinner from '../../components/LoadingSpinner'
+import checkAdminRights from '../../graphql/queries/checkAdminRights'
+import Layout from '../../components/Layout'
+import getLessons from '../../graphql/queries/getLessons'
+
+const AdminLessons: React.FC = () => {
+  const { loading, error, data } = useQuery(getLessons)
+  const [lessons, setLessons] = useState(null)
+  const [selectedLesson, setSelectedLesson] = useState(0)
+
+  return (
+    <Layout>
+      {loading && <LoadingSpinner />}
+      {error && <h1>Error</h1>}
+      {data && (
+        <div className="row mt-4">
+          <AdminLessonsSideBar
+            setLessons={setLessons}
+            lessons={lessons || data.lessons}
+            setSelectedLesson={setSelectedLesson}
+          />
+          <AdminLessonInfo
+            setLessons={setLessons}
+            lessons={lessons || data.lessons}
+            selectedLesson={selectedLesson}
+          />
+        </div>
+      )}
+    </Layout>
+  )
+}
+
+const Lessons: React.FC = () => {
+  const { loading, error, data } = useQuery(checkAdminRights)
```

Comment:
rather than creating a new query `adminRights`, you can use `sessien.user.isAdmin` flag